src/components/button/button-close.spec.js

```diff
@@ -1,105 +1,128 @@
-import { loadFixture, testVM } from '../../../tests/utils'
+import ButtonClose from './button-close'
+import { mount } from '@vue/test-utils'
 
 describe('button-close', () => {
-  beforeEach(loadFixture(__dirname, 'button-close'))
-  testVM()
-
-  it('default should have class close', async () => {
-    const {
-      app: { $refs }
-    } = window
-    expect($refs.default).toHaveClass('close')
+  it('has root element "button"', async () => {
```

Comment:
@tmorehouse we should drop `async` here too while we are at it.

Reply:
This one allows multiple "it" tests to run in parallel, and actually supports async (especially needed if we use `await $nextTick`).

`describe`, per the Jest docs (and the new warnings it spews, after the last upgrade today), doesn't support async modifier.